test_command_line_tool.d

```diff
@@ -12,13 +12,35 @@ int main(string[] args)
     try { return tryMain(args); }
     catch(SilentException) { return 1; }
 }
+
 int tryMain(string[] args)
 {
-    foreach (entry; dirEntries("test", "*.har", SpanMode.shallow))
+    // TODO: move to std.path
+    version(Windows)
+        string exeExtention=".exe";
+    else
+        string exeExtention;
+
+    auto rootDir=__FILE_FULL_PATH__.dirName;
+    auto outDir = rootDir.buildPath("out");
+    auto harExe = outDir.buildPath("har"~exeExtention);
+
+    // TODO: move this pattern to std.file using RAII
+    auto oldDir=getcwd;
+    scope(exit)
+        chdir(oldDir);
+    chdir(rootDir);
```

Comment:
I'd rather not change directory, let's just make filenames relative if we can

Reply:
done